reportportal_client/service.py

```diff
@@ -139,6 +139,17 @@ def finish_launch(self, end_time, status=None):
         logger.debug("finish_launch - Stack: %s", self.stack)
         return _get_msg(r)
 
+    def stop_launch(self, end_time, status=None):
```

Comment:
`stop_launch` is almost the same as `finish_launch`  except `s/finish_/stop_/'.
It would be grate to create a common method like `_finalize_launch` or any other name you like with semantics `_finalize_launch(self, end_time, status=None, method="finish")`. But it is required to check clients which use this lib because of semantics changes. What do you think about that?

Reply:
@krasoffski I think that it is better to follow the original API while doing such packages to work with it. Mapping method to method looks much more clear than some mixed method doing multiple things. So the idea not in functionality of the package itself but in following functionality of the base product.

So here I would stay this code as it is now.